Core/OfficeDevPnP.Core/Framework/Provisioning/ObjectHandlers/ObjectNavigation.cs

```diff
@@ -185,6 +180,48 @@ public override TokenParser ProvisionObjects(Web web, ProvisioningTemplate templ
 
         #region Utility methods
 
+        private bool WebSupportsProvisionNavigation(Web web, ProvisioningTemplate template)
+        {
+            if (template.Navigation == null) return true;
```

Comment:
Hi Mikael,
Just one question about line 185. I'm wondering why you have added this check, considering that having the template.Navigation equal to null doesn't mean that the site supports the provisioning of navigation. Moreover in line 95 we already check for (template.Navigation != null) so this check is somehow redundant. What am I missing? Thanks.